src/containers/ConfirmPassword/confirmPass.jsx

```diff
@@ -33,85 +34,71 @@ class ConfirmPass extends Component {
             values.password
           );
           localStorage.setItem('confirm', true);
-          await push(EDIT_ACCOUNT);
+          await handlePush(EDIT_ACCOUNT);
         } catch (error) {
-          this.setState({ errorMesage: 'Invalid password' });
+          if (error.message) handleErrorMessage(error.message);
```

Comment:
I think you should check for the error not the message 
```suggestion
          if (error) handleErrorMessage(error.message);
```